src/main/java/com/thebois/views/GameScreen.java

```diff
@@ -47,6 +47,10 @@ public GameScreen(final ShapeRenderer batch,
         this.views = views;
     }
 
+    public IProjector getProjector() {
+        return new ViewportWrapper(viewport);
+    }
```

Comment:
No javadoc?

Reply:
Hmm, checkstyle isn't picking that up, which is really strange considering it's not in the suppression XML file. Will add it.

Reply:
Just realized that getters and setters don't need javadoc. Nevermind. Maybe this one should be an exception though?

Reply:
I agree with you, I'm adding javadoc for it.